packages/wonder-blocks-progress-spinner/src/components/__docs__/circular-spinner.stories.js

```diff
@@ -0,0 +1,181 @@
+    component: CircularSpinner,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    decorators: [
+        (Story: any): React.Element<typeof View> => (
+            <View style={styles.example}>
+                <Story />
+            </View>
+        ),
+    ],
+};
+
+export const Default: StoryComponentType = (args) => (
+    <CircularSpinner {...args} />
+);
+
+export const Sizes: StoryComponentType = () => (
+    <table>
+        <tr>
+            <td>
+                <LabelLarge>xsmall</LabelLarge>
+            </td>
+            <td>
+                <LabelLarge>small</LabelLarge>
+            </td>
+            <td>
+                <LabelLarge>medium</LabelLarge>
+            </td>
+            <td>
+                <LabelLarge>large</LabelLarge>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <CircularSpinner size={"xsmall"} style={styles.distanced} />
+            </td>
+            <td>
+                <CircularSpinner size={"small"} style={styles.distanced} />
+            </td>
+            <td>
+                <CircularSpinner size={"medium"} style={styles.distanced} />
+            </td>
+            <td>
+                <CircularSpinner size={"large"} style={styles.distanced} />
+            </td>
+        </tr>
+        <tr className={css(styles.darkBackground)}>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"xsmall"}
+                    style={styles.distanced}
+                />
+            </td>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"small"}
+                    style={styles.distanced}
+                />
+            </td>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"medium"}
+                    style={styles.distanced}
+                />
+            </td>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"large"}
+                    style={styles.distanced}
+                />
+            </td>
+        </tr>
+    </table>
+);
+
+Sizes.parameters = {
+    docs: {
+        storyDescription: `The available sizes for progress spinner are
+            \`"xsmall"\`, \`"small"\`, \`"medium"\`, and \`"large"\`.
+            This is set with the \`size\` prop.`,
+    },
+};
+
+export const Light: StoryComponentType = () => (
+    <View style={styles.darkBackground}>
+        <CircularSpinner light={true} />
+    </View>
+);
+
+Light.parameters = {
+    docs: {
```

Comment:
**thought:** You could also set the Storybook's `darkBlue` background by including it as a key here:

```suggestion
    backgrounds: {
        default: "darkBlue",
    },
    docs: {
```

By doing this, the whole canvas takes that `darkBlue` bg.

(Sorry if I missed this in previous reviews, but just remembered that).

Reply:
OOOOOOOO I like that!